src/Home/Services.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styles from "./style.module.css";
+import { serviceData } from "../constants";
+
+const Services = () => {
+  return (
+    <>
+      <section className={styles.serviceContainer}>
+        <h2>Learn More About Our Services</h2>
+        <div className={styles.cardContainer}>
+          {serviceData.map((service) => (
```

Comment:
what if `serviceData` is undefined here, it will result in WSOD. What can we do to prevent such errors in production?